common/info.ts

```diff
@@ -55,4 +55,11 @@ export interface AllState {
     users: IUser[];
     reviews: IReview[];
     movies: IMovie[];
+}
+
+export interface ITMDBMovie {
```

Comment:
IT ?

Reply:
I pour interface et apres TMDB pour the movie database